test/wrap_test.rb

```diff
@@ -36,7 +36,7 @@ class SoftcoreBand < HardcoreBand
       let(:format) { format }
 
       representer!(:module => mod) do
-        self.representation_wrap = lambda { |args| "#{name}#{args[:number]}" }
+        self.representation_wrap = lambda { |number:, **| "#{name}#{number}" }
```

Comment:
@apotonick I changed only few test expectations like this and one more to use kwargs, as there were none. 

https://github.com/trailblazer/representable/pull/240/files#diff-367119c875f06023f45c03f9dfefa7cf6d6dc761c50746e60e464f6548904cf1R6